tests/__init__.py

```diff
@@ -109,6 +109,9 @@ def from_mm(dpi, x=0, y=0):
 
 
 class TestBase(object):
+    def teardown(self):
+        sleep(3)
```

Comment:
The other option is haveing your tests fail 7 out of 10, which  is unacceptable as well. At least this way we will have more or less reliable tests in the CI, instead of having to restart the CI manually each time it unexpectedly fails. This already happens in libratbag test suite, and those test are much more reliable than this ones, still they are a pain to restart manually.

The only way to properly fix this is `hidtools` but I guess that won't happen anytime soon.

Reply:
well, no. Adding such a bad workaround is definitively not acceptable. Please bug the hid-tools maintainers to fix their bugs instead of papering over it.

Reply:
I already opened a bug report. So should I just remove it and have unusable tests for now?